crypto/libsodium-fork/src/libsodium/crypto_core/ed25519/ref10/ed25519_ref10.c

```diff
@@ -803,6 +803,246 @@ ge25519_double_scalarmult_vartime(ge25519_p2 *r, const unsigned char *a,
     }
 }
 
+/*
+ r = - a * A + b * B
+ where a = a[0]+256*a[1]+...+256^31 a[31].
+ and b = b[0]+256*b[1]+...+256^31 b[31].
+ B is the Ed25519 base point (x,4/5) with x positive.
+
+ Only used for VRF verification.
+
+ Same as ge25519_double_scalarmult_vartime
+ but with a minus "-" before a
+ */
+
+void
+ge25519_double_sub_scalarmult_vartime(ge25519_p2 *r, const unsigned char *a,
```

Comment:
I guess I'm not a fan of making commits to a sort of reference copied library with our own improvements.  I wonder if it could be accomplished in a different compilation unit, in a different static library or maybe a header.